docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,421 @@ aliases:
 
 # Brigade Gateways
 
-This guide explains how gateways work, and provides guidance for creating your own
-gateway.
+This guide explains how gateways work, and provides guidance for creating your
+own.
 
 ## What Is A Brigade Gateway?
 
-The [Brigade architecture](design.md) is oriented around the concept that Brigade
-scripts run as a response to one or more events. In Brigade, a _gateway_ is an
-entity that generates events. Often times, it translates some external trigger
-into a Brigade event.
+The [Brigade architecture](/topics/design) is oriented around the concept that
+Brigade scripts run as a response to one or more events. In Brigade, a gateway
+is an entity that generates events. Often times, it translates some external
+trigger into a Brigade event.
```

Comment:
Any thoughts about backing off from the imprecise "often?" With a slightly narrower definition of gateway, this is _always_ true.

Suggestion:

> The [Brigade architecture](/topics/design) is oriented around the concept that Brigade scripts run as a response to one or more events. In Brigade, a gateway is a component that receives triggers from some external system and translates them into Brigade events.